gcp/modules/gcp-stackdriver-lbm/resources/log_based_metrics/iam_modify.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "iam.modify",
+  "filter": "(resource.type=\"project\" AND protoPayload.methodName=\"SetIamPolicy\") OR (resource.type=\"service_account\" AND protoPayload.methodName:\"SetIAMPolicy\") AND protoPayload.authenticationInfo.principalEmail!=\"${common_sa}\""
```

Comment:
Is this expression equivalent to `(A OR B) AND C` or `A OR (B AND C)`?  I assume the first one, but maybe adding parenthesis would make it more obvious here?

Reply:
Yep, missing brackets, thanks!